nginx-directory/configfile.libsonnet

```diff
@@ -11,8 +11,10 @@ function(services) {
         return 302 %(url)s;
       ||| % service
       else |||
-        proxy_pass      %(url)s$2$is_args$args;
-        proxy_set_header    Host $host;
+        #automagically replaces non-regex location identifier with given proxy_pass url.
+        proxy_pass          %(url)s;
+        #specifying redirect with scheme and http_host because the "default" option doesn't persist the same port.
```

Comment:
If I understand the `proxy_redirect` documentation correctly, "default" should persist the port if our `proxy_pass` URL does not consist of NGINX variables. I guess it's safer to be explicit here because there is nothing stopping a consumer of this library using NGINX variables in the interpolated `url` variable.

Reply:
I wonder if the disconnect was due to my port was being forwarded from localhost:8080:
`kubectl port-forward --namespace darren-test deployment/nginx 8080:80`.

With the old setup, and various iterations working on this, redirects from my local address (e.g., `localhost:8080/bacon/some_directory`) would redirect to the following:
- `localhost/some_directory/`,
- `localhost/bacon/some_directory/`, and once I think
- `localhost:8080/some_directory/`. 
 
Being this explicit seemed to be "just right".

Reply:
I agree, I don't see any problems with being explicit in this case.